packages/@react-spectrum/color/test/ColorField.test.js

```diff
@@ -150,7 +150,7 @@ describe('ColorField', function () {
     // should call onChange when input is cleared
     expect(colorField.value).toBe('');
     expect(onChangeSpy).toHaveBeenCalledTimes(1);
-    expect(onChangeSpy).toHaveBeenCalledWith(null);
+    expect(onChangeSpy).toHaveBeenCalledWith(undefined);
```

Comment:
won't this cause a problem because it will switch between controlled and uncontrolled?